resources/ceph_file_system.py

```diff
@@ -14,49 +12,16 @@
 
 class CephFileSystem(BaseOCSClass):
     """
-    Cephfilesystem kind class.
+    Cephfilesystem kind class
     Provides basic methods to operate on Cephfilesystem kind.
     """
 
     def __init__(self, **kwargs):
         """
-        Initializer function
+        Initializer method
 
         kwargs:
             Copy of ocs/defaults.py::CEPHFILESYSTEM dictionary
         """
-        self.fs_data = default.CEPHFILESYSTEM_DICT
-        self.fs_data.update(kwargs)
-        super(CephFileSystem, self).__init__(
-            self.fs_data['apiVersion'], self.fs_data['kind'],
-            self.fs_data['metadata']['namespace']
-        )
-
-        self.temp_yaml = tempfile.NamedTemporaryFile(
-            mode='w+', prefix='CEPHFS_', delete=False
-        )
-        self._name = self.fs_data['metadata']['name']
-
-    @property
-    def name(self):
-        return self._name
-
-    def reload(self):
-        self.fs_data = load_yaml_to_dict(self.temp_yaml.name)
-
-    def get(self, resource_name):
-        return self.ocp.get(resource_name=resource_name)
-
-    def create(self):
-        log.info(f"Adding CephFS with name {self.name}")
-        with open(self.temp_yaml.name, 'w') as yaml_file:
-            yaml.dump(self.fs_data, yaml_file)
-        return self.ocp.create(yaml_file=self.temp_yaml.name)
-
-    def delete(self):
-        self.ocp.delete(resource_name=self.name)
-
-    def apply(self, **data):
-        with open(self.temp_yaml.name, 'w') as yaml_file:
-            yaml.dump(data, yaml_file)
-        assert self.ocp.apply(yaml_file=self.temp_yaml.name)
+        self.data = kwargs
+        super(CephFileSystem, self).__init__(**kwargs)
```

Comment:
Dont we still need the Kind, Metadata and namespace to be static here for each kind?